tokio/src/runtime/scheduler/multi_thread/worker.rs

```diff
@@ -479,6 +479,7 @@ impl Context {
                     None => return Ok(core),
                 };
 
+                coop::consume_one();
```

Comment:
This could use a describing comment for future reference. It's hard to see why this was added out of context.